analysis/study_definition.py

```diff
@@ -29,9 +29,9 @@
     haem_cancer_codes,
     lung_cancer_codes,
     other_cancer_codes,
-    creatinine_codes,
-    renal_replacement_codes,
     dialysis_codes,
+    kidney_transplant_codes,
```

Comment:
You can use `from codelists import *` to import everything from `codelists.py` and avoid having to type these all out

Reply:
Thx! Did this on purpose for understanding my code better - will probably change to the wild card in future repo's!